rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -105,6 +107,36 @@ object SystemProcesses {
           } yield ()
       }
 
+      def getErrorMessagePar(str: String) =
+        RevAddress
+          .parse(str)
+          .swap
+          .toOption
+          .fold(Par.defaultInstance)(GString(_))
```

Comment:
let's not use `deafaultInstance` and just `Par()` instaed. The former is somewhat protobuf-specific

Reply:
weren't we supposed to return an "Either"? Guess you do that in latter commits?

Reply:
The Either makes sense for a parse function. The validate returns an error message if there was a problem or a Nil if the validation was successful